packages/graphql-transformer-core/src/util/transformConfig.ts

```diff
@@ -254,20 +259,40 @@ export async function loadProject(projectDirectory: string, opts?: ProjectOption
  * Preference is given to the `schema.graphql` if provided.
  * @param projectDirectory The project directory.
  */
-export async function readSchema(projectDirectory: string): Promise<string> {
-  const schemaFilePath = path.join(projectDirectory, 'schema.graphql');
+export async function readSchema(projectDirectory: string): Promise<{schema: string, modelToDatasourceMap: Map<string, DatasourceType>}> {
+  let modelToDatasourceMap = new Map<string, DatasourceType>();
+  const schemaFilePaths = [
+    path.join(projectDirectory, 'schema.graphql'),
+    path.join(projectDirectory, 'schema.rds.graphql')
+  ];
+
+  const existingSchemaFiles = schemaFilePaths.filter( path => fs.existsSync(path));
   const schemaDirectoryPath = path.join(projectDirectory, 'schema');
-  const schemaFileExists = await fs.exists(schemaFilePath);
-  const schemaDirectoryExists = await fs.exists(schemaDirectoryPath);
-  let schema;
-  if (schemaFileExists) {
-    schema = (await fs.readFile(schemaFilePath)).toString();
-  } else if (schemaDirectoryExists) {
-    schema = (await readSchemaDocuments(schemaDirectoryPath)).join('\n');
+
+  let schema = "";
+  if (!(_.isEmpty(existingSchemaFiles))) {
+    // Schema.graphql contains the models for DynamoDB datasource
+    // Schema.rds.graphql contains the models for imported 'MySQL' datasource 
+    // Intentionally using 'for ... of ...' instead of 'object.foreach' to process this in sequence 
+    for (const file of existingSchemaFiles) {
+      const datasourceType = file.endsWith('.rds.graphql') ? constructDataSourceType("MySQL", false) : constructDataSourceType("DDB");
+      const fileSchema = (await fs.readFile(file)).toString();
+      modelToDatasourceMap = new Map([...modelToDatasourceMap.entries(), ...constructDataSourceMap(fileSchema, datasourceType).entries()]);
+      schema += fileSchema;
+    }
+  } else if (fs.existsSync(schemaDirectoryPath)) {
+    // Schema folder is used only for DynamoDB datasource
+    const datasourceType = constructDataSourceType("DDB");
+    const schemaInDirectory = (await readSchemaDocuments(schemaDirectoryPath)).join('\n');
+    modelToDatasourceMap = new Map([...modelToDatasourceMap.entries(), ...constructDataSourceMap(schemaInDirectory, datasourceType).entries()]);
+    schema += schemaInDirectory;
   } else {
-    throw new Error(`Could not find a schema at ${schemaFilePath}`);
+    throw new Error(`Could not find a schema at ${schemaFilePaths[0]}`);
```

Comment:
If the `schemaFilePaths` is empty, this is bound to throw!

Reply:
SchemaFilePath array can't be empty, we add the api resource path just few lines before this logic.